ragna/deploy/_ui/modal_configuration.py

```diff
@@ -297,41 +315,122 @@ def toggle_card(event):
                 toggle_button.name = toggle_button.name.replace("▶", "▼")
 
         toggle_button = pn.widgets.Button(
-            name="Advanced Configurations   ▶",
+            name="Advanced Configuration of Assistants and Source Storages   ▶",
             button_type="light",
             css_classes=["modal_configuration_toggle_button"],
         )
-
         toggle_button.on_click(toggle_card)
 
-        toggle_button.js_on_click(
```

Comment:
Removed as this is broken on `main` and I couldn't get it to work (see previous comments).